### PR TITLE
Add missing ingots/bricks to ingot former

### LINKED ISSUE #1408: NuclearCraft Ingot Former has a very limited set of recipes.
**Modpack Version**
1.71

**Issue**
The Ingot Former has a very limited set of alloys and metals that it can make, compared to the Tinker's Smeltery and casting system. 
 I believe Ingot Former and Smelter recipes share a similar recipe format, and they both have CT Support, so exporting a list from one to the other doesn't seem too difficult a fix, though I may be mistaken.

### PR DESCRIPTION
**Pull request by pjc21**
Added the below ingots/bricks to Nuclearcraft ingot Former

- Brick
- Starmetal Ingot
- Demon Metal Ingot
- Enchanted Ingot
- Evil Infused Iron Ingot
- Fire Dragonsteel Ingot
- Ice Dragonsteel Ingot
- Pink Slime Ingot
- Refined Obsidian Ingot
- Refined Glowstone Ingot
- Base Essence Ingot
- Psimetal Ingot
- Fiery Ingot
- Seared Brick
- Mud Brick
- Elementium Ingot

(Collaborator's note: This closes #1408)

### CODE REVIEW

**Comment by NillerMedDild** [comment]
Thanks a ton for the PR, I really appreciate it - It looks good, but one of the contributors or myself will need to have a look in-game before accepting it :)

File: scripts/NuclearCraft.zs [hunk 1]
```diff
@@ -46,6 +46,40 @@ print("--- loading NuclearCraft.zs ---");
...
+	//Refined Glowstone Ingot
+	mods.nuclearcraft.ingot_former.addRecipe([<liquid:refinedglowstone>*144, <mekanism:ingot:3>, 1.0, 1.0]);
+	//Base Essence Ingot
+	mods.nuclearcraft.ingot_former.addRecipe([<liquid:base_essence>*144, <mysticalagriculture:crafting:32>, 1.0, 1.0]);
+	//Psimetal Ingot
+	mods.nuclearcraft.ingot_former.addRecipe([<liquid:psimetal>*144, <psi:material:1>, 1.0, 1.0]);
+	//Fiery Ingot
+	mods.nuclearcraft.ingot_former.addRecipe([<liquid:fierymetal>*144, <twilightforest:fiery_ingot>, 1.0, 1.0]);
+	//Seared Brick
+	mods.nuclearcraft.ingot_former.addRecipe([<liquid:stone>*144, <tconstruct:materials>, 1.0, 1.0]);
```
Comment (MaxNeedsSnacks) [inline scripts/NuclearCraft.zs]: This should be 72mb to match the Tinkers' Construct recipe.

image

**Comment by MaxNeedsSnacks** [comment]
This looks ready-to-merge except for that one minor detail I found ^^

File: scripts/NuclearCraft.zs (hunk 1 above)
Comment (pjc21) [reply scripts/NuclearCraft.zs]: Sorry, missed that. I just kind of assumed that all ingots were 144mb. Changed and fixed.

**Pull request merged**